JitsiConference.js

```diff
@@ -937,6 +995,8 @@ JitsiConference.prototype.onMemberJoined = function(jid, nick, role, isHidden) {
             this.updateDTMFSupport();
         },
         error => logger.error(error));
+
+    this._startStopP2PSession();
```

Comment:
gotcha, at this time i thought `_startStopP2PSession()` was starting and stopping (restarting) the session.